packages/emotion-theming/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "emotion-theming",
+  "version": "1.0.0",
+  "description": "A CSS-in-JS theming solution, forked from iamstarkov/theming.",
+  "main": "dist/cjs",
+  "module": "dist/esm",
+  "files": [
+    "src",
+    "dist"
+  ],
+  "scripts": {
+    "clean": "rimraf dist",
+    "build": "run-s clean babel:*",
+    "babel:cjs": "cross-env BABEL_ENV=cjs babel src -d dist/cjs",
+    "babel:esm": "cross-env BABEL_ENV=esm babel src -d dist/esm"
```

Comment:
Could you change this to use the rollup setup we have for the other packages?

https://github.com/emotion-js/emotion/blob/master/packages/preact-emotion/package.json#L12-L14